docs/sftt/sftt-api-spec.md

```diff
@@ -373,23 +403,17 @@ No request body.
 
 ```json
 {
-  "applicants": [
-    {
-      "userId": INT,
-      "username": STRING,
-    },
+  "users": {
+    STRING: "PENDING",
+    STRING: "PENDING",
     ...
-  ]
+  },
 }
 ```
 
 ##### `400 BAD REQUEST`
 
-If the team id is invalid.
-
-##### `401 UNAUTHORIZED`
-
-If the user is not a team leader
+If the `team_id` specified is not associated with an existing team.
```

Comment:
❓ So anyone can get the applicants? Is that what we want?

Reply:
Yeah that's what it is on master but I agree it seems weird, I'm adding that to the backend PR